.github/workflows/performance.yml

```
@@ -0,0 +1,637 @@
+    branches: [main]
+    types: [opened, synchronize]
+  schedule:
+    # Run performance tests daily at 3 AM UTC
+    - cron: '0 3 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+    inputs:
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)